ts/components/LabelledItem/LabelledItemIconOrImage.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import {
+  Image,
+  ImageSourcePropType,
+  ImageStyle,
+  StyleSheet
+} from "react-native";
+import { IOColors, IOIcons, Icon } from "@pagopa/io-app-design-system";
+import { isString } from "lodash";
+import ButtonDefaultOpacity from "../ButtonDefaultOpacity";
+
+const styles = StyleSheet.create({
+  button: {
+    paddingTop: 0,
+    paddingBottom: 0,
+    paddingLeft: 0,
+    paddingRight: 0,
+    alignSelf: "auto",
+    height: "auto"
+  }
+});
+
+type LabelledItemIconOrImageProps = {
+  icon: IOIcons | ImageSourcePropType;
+  iconColor?: IOColors;
+  imageStyle?: ImageStyle;
+  accessible?: boolean;
+  accessibilityLabelIcon?: string;
+  onPress?: () => void;
+};
+
+export const LabelledItemIconOrImage: React.FC<LabelledItemIconOrImageProps> =
+  ({
+    icon,
+    iconColor,
+    imageStyle,
+    accessible,
+    onPress,
+    accessibilityLabelIcon
+  }): React.ReactElement => (
+    <ButtonDefaultOpacity
+      testID="ButtonDefaultOpacity"
+      onPress={onPress}
+      transparent
+      accessible={accessible === undefined ? true : accessible}
```

Comment:
```suggestion
      accessible={accessible ?? true}
```
I believe the nullish coalescing operator makes it an easier read on the eye; do yo agree?